tests_integration_with_local_chain/markets/omen/test_stealing_on_markets.py

```
@@ -0,0 +1,214 @@
+
+    # Buy YES tokens from accout B (attacker) -- removing profit from any real agent that'd like to complete the job.
+    buy_yes_for_b = xdai_type(5)
+    binary_omen_buy_outcome_tx(
+        api_keys_B,
+        buy_yes_for_b,
+        agent_market,
+        binary_outcome=True,
+        auto_deposit=True,
+        web3=local_web3,
+    )
+    balance_after_buying_A, balance_after_buying_B = (
```

Ruff

Local variable `balance_after_buying_A` is assigned to but never used

Remove assignment to unused variable `balance_after_buying_A`

(F841)